ci/gpu/build.sh

```diff
@@ -64,6 +64,11 @@ gpuci_mamba_retry install -c conda-forge -c rapidsai -c rapidsai-nightly -c nvid
       "rapids-doc-env=${MINOR_VERSION}.*" \
       "shap>=0.37,<=0.39"
 
+gpuci_conda_retry remove -c conda-forge -c rapidsai -c rapidsai-nightly -c nvidia \
+      --force rapids-build-env rapids-notebook-env
+gpuci_mamba_retry install -c conda-forge -c rapidsai -c rapidsai-nightly -c nvidia \
+      treelite=2.2.1
+
 # https://docs.rapids.ai/maintainers/depmgmt/
 # gpuci_mamba_retry remove --force rapids-build-env rapids-notebook-env
```

Comment:
```suggestion
# gpuci_conda_retry remove --force rapids-build-env rapids-notebook-env
```


Since you're touching this file, can you update the comment here to use `gpuci_conda_retry remove`? This is because `gpuci_mamba_retry remove` has issues as outlined below.

https://docs.rapids.ai/maintainers/depmgmt/#testing-a-dependency

![image](https://user-images.githubusercontent.com/7400326/150007801-12072ceb-83ec-401d-b295-f3570b0df78a.png)

Reply:
@ajschmidt8 Done. Let’s merge this (pending CI) so that other PRs don’t get blocked.